src/librustc/ty/mod.rs

```diff
@@ -1845,6 +1865,119 @@ impl<'a, 'gcx, 'tcx> AdtDef {
         debug!("sized_constraint_for_ty({:?}) = {:?}", ty, result);
         result
     }
+
```

Comment:
I feel we should merge this function with `sized_constraint` and soon `dyn_sized_constraint`.

Reply:
See comment about merging this.

Reply:
I'd like to delay that change until both `Move` and `DynTrait` PRs are merged.